applications/matter_bridge/sample.yaml

```diff
@@ -26,3 +26,9 @@ tests:
     integration_platforms:
       - nrf7002dk_nrf5340_cpuapp
     platform_allow: nrf7002dk_nrf5340_cpuapp
+  applications.matter_bridge.smp_dfu.br_ble:
```

Comment:
* "debug" configuration will be redundant, I think it will be enough to have only "smp_dfu" similar to what we have for door lock
* in qurantine-integration remove 'applications.matter_bridge.release.br_ble' and for the comment in line 5 mark that excluded configuration is `applications.matter_bridge.release` that is put line there like here https://github.com/nrfconnect/sdk-nrf/blob/main/applications/matter_weather_station/sample.yaml#L14